Source/Scene/PostProcessAmbientocclusionStage.js

```diff
@@ -0,0 +1,176 @@
+define([
```

Comment:
This file is a placeholder for the AO in https://github.com/AnalyticalGraphicsInc/cesium/pull/5673